### PR TITLE
overview.md: Update to a modern password policy

### PR DESCRIPTION
**Pull request by philipandag**
https://github.com/Dasharo/edk2/pull/152

### CODE REVIEW

File: docs/dasharo-menu-docs/overview.md [hunk 1]
```diff
@@ -129,11 +129,12 @@ possible feature extension of your platform.
 
 User Password Management menu allows one to set firmware setup admin password:
 
-![](/images/menus/password_mgr.jpeg){ class=center" }
+![](/images/menus/password_mgr.jpeg){ class="center" }
 
-The password must be a strong one. The requirements are described on the help
-section on the right side of the menu page. If the password is too weak it will
-not be accepted. Also the new password must be different from the last 5
+The password requirements are compliant to modern standards. The password
```
Comment (macpijan) [inline docs/dasharo-menu-docs/overview.md:134]: any reference on these standards?

Comment (philipandag) [reply docs/dasharo-menu-docs/overview.md]: https://pages.nist.gov/800-63-3/sp800-63b.html  Appendix A sums it up pretty well. I have added a link to the site.

**Comment by mkopec** [comment]
@philipandag Looks good overall, but can you try to resolve CI failures?

**Review by mkopec — APPROVED** [review]
Feel free to merge once all CI checks pass

**Pull request merged**